test/e2e/adminUI/pages/app.js

```diff
@@ -18,19 +18,6 @@ module.exports = {
 		// APP LEVEL MENU
 		homeIcon: '.primary-navbar [data-section-label="octicon-home"]',
 		homeIconLink: '.primary-navbar [data-section-label="octicon-home"] a',
-		accessMenu: '.primary-navbar [data-section-label="Access"]',
-		fieldsMenu: '.primary-navbar [data-section-label="Fields"]',
-		booleansFieldsSubmenu: '.secondary-navbar [data-list-path="booleans"]',
-		codesFieldsSubmenu: '.secondary-navbar [data-list-path="codes"]',
-		datesFieldsSubmenu: '.secondary-navbar [data-list-path="dates"]',
-		emailsFieldsSubmenu: '.secondary-navbar [data-list-path="emails"]',
-		namesFieldsSubmenu: '.secondary-navbar [data-list-path="names"]',
-		selectsFieldsSubmenu: '.secondary-navbar [data-list-path="selects"]',
-		textsFieldsSubmenu: '.secondary-navbar [data-list-path="texts"]',
-		textareasFieldsSubmenu: '.secondary-navbar [data-list-path="textareas"]',
-		urlsFieldsSubmenu: '.secondary-navbar [data-list-path="urls"]',
-		frontPageIcon: '.primary-navbar [data-section-label="octicon-globe"]',
-		frontPageIconLink: '.primary-navbar [data-section-label="octicon-globe"] a',
```

Comment:
It seems I didn't resolve the merge conflicts very well in #2709. My bad... sorry.

This diff resolves the conflicts properly, along with the changes to the date files.